api/lib/infrastructure/serializers/jsonapi/organization-invitation-serializer.js

```diff
@@ -4,7 +4,15 @@ module.exports = {
 
   serialize(invitations) {
     return new Serializer('organization-invitations', {
-      attributes: ['organizationId', 'organizationName', 'email', 'status', 'updatedAt'],
+      transform(record) {
+        record.organization = { id: record.organizationId };
+        return record;
```

Comment:
Mieux vaut éviter de muter l'objet passé :

```suggestion
        return { ...record, organization: { id: record.organizationId } };
```

Reply:
Ce ce que je comptais faire afin d'éviter un reload() des `organization-invitations` suite à la création, cependant la création d'invitations permet d'en créer plusieurs à la fois en renseignant plusieurs e-mail dans le champ. Or, JSON:API et ember-data ne gèrent pas la création de plusieurs ressources suite à l'appel d'un POST...

Reply:
Je ne vois pas le rapport avec le fait de ne pas modifier l'objet passé ?

Reply:
On peut toujours le faire c'est vrai, mais je pensais que ça me permettrait de ne pas reload la collection d'invitations. Je vais le rajouter.

Reply:
Ça n'a rien à voir avec le reload côté client, ici on parle juste de muter ou pas un objet dans l'implémentation de l'API mais la réponse HTTP est la même.

Reply:
Ou sinon un Object.assign({}, record, obj), les objets en Js sont mutables et pour éviter de modifier l'objet d'origine, on renvoie plutôt une copie de l'objet. Merci pour ce retour, même si cette modification est obsolète.

Reply:
cette modification étant obsolète => le fichier n'est plus modifié par le ticket